natural-language-understanding/custom_classifications_example.ipynb

```diff
@@ -246,251 +267,204 @@
+     "name": "stdout",
      "text": [
-      "Successfully analyzed request. Response from NLU:\n",
-      "\n",
+      "Analysis response from recently trained model:\n",
       "{\n",
+      "    \"usage\": {\n",
+      "        \"text_units\": 1,\n",
+      "        \"text_characters\": 36,\n",
+      "        \"features\": 0\n",
+      "    },\n",
+      "    \"language\": \"en\",\n",
       "    \"classifications\": [\n",
       "        {\n",
-      "            \"class_name\": \"temperature\",\n",
-      "            \"confidence\": 0.562519\n",
+      "            \"confidence\": 0.562519,\n",
+      "            \"class_name\": \"temperature\"\n",
       "        },\n",
       "        {\n",
-      "            \"class_name\": \"conditions\",\n",
-      "            \"confidence\": 0.433996\n",
+      "            \"confidence\": 0.433996,\n",
+      "            \"class_name\": \"conditions\"\n",
       "        }\n",
-      "    ],\n",
-      "    \"language\": \"en\",\n",
-      "    \"usage\": {\n",
-      "        \"features\": 0,\n",
-      "        \"text_characters\": 36,\n",
-      "        \"text_units\": 1\n",
-      "    }\n",
+      "    ]\n",
       "}\n"
      ]
     }
    ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
    "source": [
-    "######### Create request #########\n",
-    "\n",
-    "analyze_request_data = {\n",
-    "        \"text\":\"What is the expected high for today?\",\n",
-    "        \"language\": \"en\",\n",
-    "        \"features\": {\n",
-    "            \"classifications\": {\n",
-    "                \"model\": model_id\n",
-    "            }\n",
-    "        }\n",
-    "}\n",
-    "\n",
-    "uri = url + '/v1/analyze'\n",
-    "\n",
-    "params = {\n",
-    "    'version': '2021-02-15'\n",
-    "}\n",
-    "\n",
-    "headers = {'Content-Type' : 'application/json'}\n",
-    "\n",
-    "######### Make a call to NLU #########\n",
+    "## 6. How to Delete a NLU Classifications Model\n",
     "\n",
-    "response = requests.post(uri,\n",
-    "                         params=params,\n",
-    "                         json=analyze_request_data,\n",
-    "                         headers=headers,\n",
-    "                         auth=(username, password),\n",
-    "                         verify=False,\n",
-    "                        )\n",
-    "\n",
-    "if response.status_code != 200:\n",
-    "    print('Failed to make request to model. Reason:')\n",
-    "    print(response.text)\n",
-    "\n",
-    "else:\n",
-    "    response_json = response.json()\n",
+    "Use the `delete_classifications_model` method to remove a specific classification model via a `model_id`. This operation can be verified by listing all the existing models with `list_classifications_models`. "
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "source": [
+    "model_to_delete = nlu.delete_classifications_model(model_id=model_id)\n",
```

Comment:
nit: would it make sense to rename this variable from `model_to_delete` to `deleted_model` ?